src/components/ModalPayment/ModalAmount/LanternForm.tsx

```diff
@@ -0,0 +1,271 @@
+            value={lucData.firstName}
+          />
+        </RowFormat>
+        <RowFormat width="20%" mobileWidth="30%">
+          <LabelText htmlFor="middle_initial">
+            {t('paymentProcessing.amount.labels.middle_initial')}
+          </LabelText>
+          <InputText
+            name="middleInitial"
+            type="text"
+            className="modalInput--input"
+            onChange={handleOnChange}
+            placeholder={t(
+              'paymentProcessing.amount.place_holder.middle_initial'
+            )}
+            value={lucData.middleInitial}
+          />
+        </RowFormat>
+        <RowFormat width="38%">
+          <LabelText htmlFor="last_name">
+            {t('paymentProcessing.amount.labels.last_name')}
+          </LabelText>
+          <InputText
+            name="lastName"
+            type="text"
+            className="modalInput--input"
+            onChange={handleOnChange}
+            placeholder={t('paymentProcessing.amount.place_holder.last_name')}
+            value={lucData.lastName}
+          />
+        </RowFormat>
+      </SingleRowFormat>
+
+      {amount >= LIGHT_UP_CHINATOWN_TIER_2_LIMIT && (
+        <>
+          <label>
+            <span>
+              {' '}
+              {t(
+                'paymentProcessing.amount.light_up_chinatown_address.disclaimer_1'
+              )}
+            </span>
+            <b>
+              {' '}
+              {t('paymentProcessing.amount.light_up_chinatown_address.address')}
+            </b>
+            <span>
+              {' '}
+              {t(
+                'paymentProcessing.amount.light_up_chinatown_address.disclaimer_2'
+              )}
+            </span>
+          </label>
+          <br />
+          <br />
+
+          <TextTrigger onClick={(e) => handleAddress(e, true)}>
+            {' '}
+            {t(
+              'paymentProcessing.amount.light_up_chinatown_address.address_button'
+            )}
+          </TextTrigger>
+          <br />
+          <br />
+
+          {showAddress && (
+            <>
+              <NameRow>
+                <RowFormat width="65%" mobileWidth="100%">
+                  <LabelText htmlFor="full_name">
+                    {t('paymentProcessing.amount.labels.full_name')}
+                  </LabelText>
+                  <InputText
+                    name="fullName"
+                    type="text"
+                    className="modalInput--input"
+                    onChange={handleOnChange}
+                    placeholder={t(
+                      'paymentProcessing.amount.place_holder.full_name'
+                    )}
+                    value={lucData.fullName}
+                  />
+                </RowFormat>
+                <TextTrigger onClick={(e) => handleAddress(e, false)}>
+                  {' '}
+                  {t(
+                    'paymentProcessing.amount.light_up_chinatown_address.cancel_address_button'
```

Comment:
We should probably handle the case where the user enters the shipping address and then decides not to have it shipped, so clear the data we have for it.